### PR TITLE
Add  2 icons on lasso sub toolbar, disable/enable based on lasso sele…

### PR DESCRIPTION
**Pull request by ramjith**
# Pull Request

Hi,

Thanks for helping us improve the Predix UI platform by submitting a Pull Request.

To help us merge your Pull Request as fast as possible, please fill out the following sections:

* ## A description of the changes proposed in the pull request:
Add  2 icons on lasso sub toolbar, disable/enable based on lasso selection for US424279

* ## A reference to a related issue (if applicable):

* ## @mentions of the person or team responsible for reviewing proposed changes:

* ## working tests:

### CODE REVIEW

**Comment by gedigital-apm** [comment]
Since the `toolbarConfig` is configurable, 
Is it possible to control everything in application itself ?

**Comment by ramjith** [comment]
> Since the `toolbarConfig` is configurable,
> Is it possible to control everything in application itself ?

It's not only just adding 2 icons, there are few functionalities that requires px-vis change like disable/enable icons based on lasso selection

File: px-vis-toolbar.html [hunk 1]
```diff
@@ -146,7 +146,7 @@
       <span id="mainRow" class$="flex [[_getRowDirection(horizontalAlignment)]]">
         <template id="mainRepeat" is="dom-repeat" items="{{_internalConfig}}" strip-whitespace>
           <span
-            class$="u-p- flex__item main-item [[_getSelectedClass(item.selected)]] [[_getDisabledClass(item.disabled)]] [[_hide(item.hidden)]]"
+            class$="u-p- flex__item main-item [[_getSelectedClass(item.selected)]] [[_getDisabledClass(item)]] [[_hide(item.hidden)]]"
```
Comment (gedigital-apm) [inline px-vis-toolbar.html:149]: Is it `_getDisabledClass(item)` or `_getDisabledClass(item.*)` ?

File: px-vis-toolbar.html [hunk 2]
```diff
@@ -162,7 +162,7 @@
         <span id="subRow" class$="flex subrow--container [[_getRowDirection(horizontalAlignment)]] [[_getSubRowClass(subToolbarAlignment)]]">
           <template id="subRepeat" is="dom-repeat" items="{{currentSubConfig}}" strip-whitespace>
             <span
-              class$="u-p- flex__item main-item subrow--item [[_getSelectedClass(item.selected)]] [[_getDisabledClass(item.disabled)]] [[_hide(item.hidden)]]"
+              class$="u-p- flex__item main-item subrow--item [[_getSelectedClass(item.selected)]] [[_getDisabledClass(item, currentSubConfig)]] [[_hide(item.hidden)]]"
```
Comment (gedigital-apm) [inline px-vis-toolbar.html:165]: Is it _getDisabledClass(item) or _getDisabledClass(item.*) ?

File: px-vis-toolbar.html [hunk 3]
```diff
@@ -398,13 +403,30 @@
...
+           }
+          },
           clearCrosshair: {
             tooltipLabel: 'Clear lasso data',
             icon: 'px-nav:close',
             onClick: function() {
               this.set('crosshairData', { 'rawData': [], 'timeStamps': []});
               this.set('interactionSpaceConfig.resetLassoData', !this.interactionSpaceConfig.resetLassoData);
               this.set('resetLassoData', !this.resetLassoData);
+              this.fire('diable-lasso-filter');
```
Comment (vibhaMalviya) [inline px-vis-toolbar.html:429]: Looks like typo in event name, should be 'disable-lasso-filter'.

File: px-vis-toolbar.html [hunk 4]
```diff
@@ -454,6 +497,7 @@
...
```
Comment (vibhaMalviya) [inline px-vis-toolbar.html:500]: same as above comment

File: px-vis-toolbar.html (hunk 2 above)
Comment (ramjith) [reply px-vis-toolbar.html:165]: As we talked the observer is for currentSubConfig

File: px-vis-toolbar.html (hunk 3 above)
Comment (ramjith) [reply px-vis-toolbar.html]: Fixed it

**Pull request merged**